test/e2e/helm_test.go

```diff
@@ -41,3 +41,40 @@ func TestHelmCrdInstallIsCreated(t *testing.T) {
 			return p.Name == "hook"
 		}))
 }
+
+func TestServiceCatalog(t *testing.T) {
+	t.Skip("slow")
```

Comment:
we skip these tests - need to figure out the correct e2e test for this - @jessesuen any suggestions?